packages/coinstac-ui/app/render/state/ducks/util.js

```diff
@@ -1,7 +1,11 @@
 import app from 'ampersand-app';
 
+export const logUI = (level, msg) => {
+  app.renderLogger[level](msg);
```

Comment:
Also, `logUI` is a bit redundant: we’re in the `render` directory, and these calls happen from modules containing user interface components. How does simply `log` sound?